entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+CA_KEY_FILE='ca.key'
+CA_CERT_FILE='ca.crt'
+
+# Generate private key for root CA
+# Options:
+#  -out : name of file to write generated key to
+#  4096 : number of bits in the key
+openssl genrsa -out $CA_KEY_FILE 4096
```

Comment:
Hello,

here we build cert at runtime (every time we launch the container)
Is it ok ?

Reply:
Hello. Yes it is exactly for what this image is designed. It should be run as Kubernetes job which generates CA private key and certificate.

Reply:
ok 👍